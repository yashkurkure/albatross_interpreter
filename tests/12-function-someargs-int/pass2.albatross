fun f void (i string) {
  printstring(i);
  return;
}

f("CS 473 UIC");
return 0;
