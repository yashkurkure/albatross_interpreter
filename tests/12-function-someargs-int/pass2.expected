CS 473 UIC
